index.js

```diff
@@ -276,6 +276,143 @@ async function getCircuit(conv) {
+      conv.contexts.set('setPresence_getPresence', 5, { 
+    });
+  }
+  return;
+});
+
+/**
+ * Set a logged on user to dnd
+*/
+app.intent('set.presence - dnd', async conv =>
+{
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+
+  const { untilTime } = conv.contexts.input['setpresence_data'].parameters;
+  const { duration } = conv.contexts.input['setpresence_data'].parameters;
+
+  await circuit.setPresenceDnd(untilTime, duration);
+  conv.ask(`Your online presence is set to Do Not Disturb. Anything else?`);
+  conv.contexts.set('anything_else', 5);
+});
+
+/**
+ * Set a logged on user to dnd
+*/
+app.intent('get.presence', async conv =>
+{
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+  const yourPresence = await circuit.getUserPresence();
+  conv.ask(`Your online presence is set to ${yourPresence}. Anything else?`);
+  conv.contexts.set('anything_else', 5);
+});
+
+/**
+ * Set a logged on user to dnd
+*/
+app.intent('get.dndTime', async conv =>
+{
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+
+  const yourPresence = await circuit.getUserPresence();
+  if(yourPresence == 'DND'){
+    const timeLeft = await circuit.getDndTime();
+    const mLeft = Math.floor((timeLeft - Date.now())/60000);//sets the time left in minutes
+
+    if(mLeft > 60){
+      conv.ask(`Your DND is set until ${Math.floor(mLeft/60)} hour(s) and ${Math.floor(((mLeft/60)- Math.floor(mLeft/60))*60)} minute(s) from now. Anything Else?`);
+      conv.contexts.set('anything_else', 5);
+    }
+    else  {
+      conv.ask(`Your DND is set until ${mLeft} minutes from now. Would there be anything else?`);
+      conv.contexts.set('anything_else', 5);
+    } 
+  }
+  else  {
+    conv.ask('It seems that you are not set to Do Not Disturb. Would you like to do anything else?');
+    conv.contexts.set('anything_else', 5);
+  }
+});
+
+/**
+ * Set a logged on user's status message
+*/
+app.intent('set.statusmessage', async conv =>
+{
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+
+  const {statusMessage} = conv.contexts.input['setstatusmessage_data'].parameters;
+
+  await circuit.setMyStatusMessage(statusMessage);
+
+  conv.ask(`Your status message is now set to '${statusMessage}'. May I do anything else for you today?`);
+  conv.contexts.set('anything_else', 5);
+});
+
+/**
+ * Set a logged on user's status message
+*/
+app.intent('get.statusmessage', async conv =>
+{
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+
+  let statusMessage = await circuit.getMyStatusMessage();
+  if(statusMessage != ''){
+    conv.ask(`Your status message is '${statusMessage}'. May I do anything else for you today?`);
+    conv.contexts.set('anything_else', 5);
+  }
+  else if(statusMessage ==''){
+    conv.ask(`It appears your status message is blank. May I do anything else for you today?`); 
```

Comment:
If it's just a single line string without templating I think you can just use ' instead of `